### PR TITLE
ℕ-induction and property of subst fot Sets

### PR DESCRIPTION
**Pull request by marcinjangrzybowski**
Those are unrelated additions, but I dont know how to split those changes into seperate pull requests.

### CODE REVIEW

File: Cubical/Foundations/Transport.agda [hunk 1]
```diff
@@ -42,3 +43,12 @@ isEquivTransport {A = A} {B = B} p =
...
 transportEquiv : ∀ {ℓ} {A B : Type ℓ} → A ≡ B → A ≃ B
 transportEquiv p = (transport p , isEquivTransport p)
+
+pathToIso : ∀ {ℓ} {A B : Type ℓ} → A ≡ B → Iso A B
+pathToIso x = iso (transport x) (transport⁻ x ) ( transportTransport⁻ x) (transport⁻Transport x)
+
+isSet-subst : ∀ {ℓ ℓ′} {A : Type ℓ} {B : A → Type ℓ′}
+                → (isSet-A : isSet A)
+                → ∀ {a}
+                → ∀ p → ∀ x → subst {x = a} B p x ≡ x
```
Comment (mortberg) [inline Cubical/Foundations/Transport.agda]: Could you please add the types to a, p and x?

File: Cubical/Foundations/Transport.agda [hunk 2]
```diff
@@ -42,3 +43,12 @@ isEquivTransport {A = A} {B = B} p =
...
+isSet-subst {B = B} isSet-A p x = subst (λ p′ → subst B p′ x ≡ x) ((isSet-A) _ _ refl p) (substRefl {B = B} x)
```
Comment (mortberg) [inline Cubical/Foundations/Transport.agda]: You don't need parentheses around isSet-A

**Comment by marcinjangrzybowski** [comment]
I made requested changes, next time i will try to put more attention to those details before filling pull request :)

**Comment by mortberg** [comment]
Thanks for your contribution!

**Pull request merged**